app/uk/gov/hmrc/uknwauthcheckerapistub/controllers/EORIStubController.scala

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmrc.uknwauthcheckerapistub.controllers
+
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Request}
+import uk.gov.hmrc.uknwauthcheckerapistub.tools.Helper
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton()
+class EORIStubController @Inject()(cc: ControllerComponents)
+  extends BackendController(cc) with Helper {
+
+  def checkEORI(): Action[AnyContent] = Action { implicit request: Request[AnyContent] =>
```

Comment:
This endpoint should replicate the request structure for EIS's endpoint. At the moment it's not taking anything.